surfer/viz.py

```diff
@@ -2175,11 +2196,18 @@ def save_movie(self, fname, time_dilation=4., tmin=None, tmax=None,
 
         Notes
         -----
-        This method requires FFmpeg to be installed in the system PATH. FFmpeg
-        is free and can be obtained from `here
-        <http://ffmpeg.org/download.html>`_.
+        Requires imageio package, which can be installed together with
+        PySurfer with::
+
+            $ pip install -U pysurfer[save_movie]
         """
-        assert_ffmpeg_is_available()
+        try:
+            import imageio
+        except ImportError:
+            raise ImportError("Saving movies from PySurfer requires the "
+                              "imageio library. To install imageio with "
+                              "PySurfer, run\n\n    $ pip install -U "
+                              "pysurfer[save_movie]\n")
```

Comment:
Why not just suggest `pip install --upgrade imageio` directly? The suggestion will do more than just satisfy the ImageIO requirement, it will also potentially upgrade PySurfer, which I don't think is the cleanest suggestion.